services/elasticache/pkg/resource/replication_group/custom_update_api_test.go

```diff
@@ -16,6 +16,8 @@ package replication_group
 import (
 	"context"
 	"fmt"
+	"github.com/aws/aws-controllers-k8s/pkg/requeue"
+	"github.com/pkg/errors"
```

Comment:
this is only a test file of course, but common practice is to put third-party imports together separated by a newline after standard library imports.